test/Microsoft.AspNetCore.Razor.Evolution.Test/TestFiles/IntegrationTests/BasicIntegrationTest/SimpleTagHelpers.ir.txt

```diff
@@ -0,0 +1,23 @@
+Document - 
+    Checksum - 
+    NamespaceDeclaration -  - 
+        UsingStatement -  - System
+        UsingStatement -  - System.Threading.Tasks
+        ClassDeclaration -  -  -  -  - 
+            DeclareTagHelperFields - 
+            RazorMethodDeclaration -  -  -  -  - 
+                HtmlContent - (0:0,0 [0] SimpleTagHelpers.cshtml) - <p>Hola</p>\n
+                TagHelper - 
+                    InitializeTagHelperStructure - 
```

Comment:
Update the test node serializer to output more information about the TagHelper bits. For example, this guy the tagmode/tagname.